src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,57 @@
+@use "../utils/var.scss";
+
+.watch {
+  width: $watch-area;
+  height: $watch-area;
+  background-color: $color-watch;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 50px;
+  border: 3px solid $color-black;
+
+  &__arrow {
+    height: $watch-long;
+    width: $watch-long;
+    border-radius: 50%;
+    background-color: $color-black;
+    z-index: 1;
```

Comment:
you may write it after `minute` and `second` in HTML instead

Reply:
![image](https://user-images.githubusercontent.com/119611480/216824473-5301b393-82cb-4aaf-aba3-90d35732e27d.png)
Yes, but it doesn't quite work